reported upstream on 24 Apr 2009:
http://lists.gnu.org/archive/html/bug-wget/2009-04/msg00042.html

off-list reply indicates fix in SVN:

<quote>
	I've added this to the issue tracker:
	https://savannah.gnu.org/bugs/?26335

	But then when I actually got a chance to look at the patch, saw how
	trivial it was. I committed something close, but not identical, to it
	(particularly for hash.c):
	http://hg.addictivecode.org/wget/mainline/rev/1f5794065912

	If that patch doesn't work right for you, please let me know, and I'll
	reopen the issue I had opened (and have now closed).
</quote>

diff -ru -x *.Po -x *.Plo wget-1.11.1.orig/src/connect.c wget-1.11.1/src/connect.c
--- wget-1.11.1.orig/src/connect.c	Thu Apr  3 13:11:16 2008
+++ wget-1.11.1/src/connect.c	Thu Apr  3 13:12:00 2008
@@ -55,6 +55,10 @@
 # include <sys/time.h>
 #endif
 
+#ifdef HAVE_STDINT_H
+# include <stdint.h>
+#endif
+
 #include "wget.h"
 #include "utils.h"
 #include "host.h"
diff -ru -x *.Po -x *.Plo wget-1.11.1.orig/src/hash.c wget-1.11.1/src/hash.c
--- wget-1.11.1.orig/src/hash.c	Thu Apr  3 13:11:16 2008
+++ wget-1.11.1/src/hash.c	Thu Apr  3 13:12:58 2008
@@ -35,6 +35,10 @@
 # include <config.h>
 #endif
 
+#ifdef HAVE_STDINT_H
+# include <stdint.h>
+#endif
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <assert.h>
